src/cljs/ataru/virkailija/editor/handlers.cljs

```diff
@@ -1620,3 +1630,22 @@
     (let [path (db/current-form-properties-path db [:closed])
           value (not (get-in db path))]
       (assoc-in db path value))))
+
+(reg-event-db
+  :editor/update-selected-property-options
+  (fn [db [_ category selected-option-ids]]
+    (let [path (vec (db/current-form-properties-path db [(keyword category)]))]
```

Comment:
onko vec tarpeellinen?

Reply:
Kyllä tuo itse asiassa tarvitaan.. Tuo seuraava assoc-in komento ei näyttäisi onnistuvan ellei path ole vektori. Tarkempaa syytä en osaa äkkiseltään sanoa.